include/jemalloc/internal/buf_writer.h

```diff
@@ -21,6 +21,13 @@ typedef struct {
 	size_t buf_size;
 	size_t buf_end;
 	bool internal_buf;
+	/*
+	 * The following fields are for advanced APIs - see explanations
+	 * below.
+	 */
+	bool allow_auto_flush;
+	size_t commit;
```

Comment:
maybe committed?

Reply:
Changed it to `checkpoint` to align with `buf_writer_set_checkpoint()`.